src/rules/package-lavamoat-allow-scripts-conforms.ts

```diff
@@ -0,0 +1,34 @@
+import { get, has } from 'lodash';
+
+import { buildRule } from './build-rule';
+import { PackageManifestSchema, RuleName } from './types';
+
+export default buildRule({
+  name: RuleName.PackageLavamoatAllowScriptsConforms,
+  description:
+    'Does lavamoat lists `@lavamoat/preinstall-always-fail: false` in allow scripts?',
```

Comment:
It just occurred to me that we include this package in the module template, but then immediately disable its functionality. That seems odd to me. I've raised a question about this in chat to see if we still need this or not.

Reply:
Okay, it seems that we still want this. 

Maybe, however, we can reword this to sound a little more natural?

``` suggestion
    'Are postinstall scripts disabled for `@lavamoat/preinstall-always-fail`?',